From 508cf475854c83b61a63d4c5f143b9a405fef77c Mon Sep 17 00:00:00 2001
From: Lars Wendler <polynomial-c@gentoo.org>
Date: Sat, 9 Jan 2021 01:21:55 +0100
Subject: [PATCH] Revert "autotools: unbreak --enable-wayland with BSD sed"

This reverts commit 78431a361bed0b6e9f1f3ebc0de8897b069b3848
as it breaks GNU sed

https://github.com/intel/libva/issues/489

Signed-off-by: Lars Wendler <polynomial-c@gentoo.org>
---
 va/wayland/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/va/wayland/Makefile.am b/va/wayland/Makefile.am
index f48afb4..705efa3 100644
--- a/va/wayland/Makefile.am
+++ b/va/wayland/Makefile.am
@@ -70,7 +70,7 @@ va_wayland_drm.c: $(protocol_source_h)
 %-client-protocol-export.c : %.xml
 	$(AM_V_GEN)$(WAYLAND_SCANNER) code < $< > $@
 %-client-protocol.c: %-client-protocol-export.c
-	$(AM_V_GEN)$(SED) -e $$'1i\\\n#include "sysdeps.h"\n' \
+	$(AM_V_GEN)$(SED) -e '1i#include "sysdeps.h"' \
 	                   -e 's@WL_EXPORT@DLL_HIDDEN@g' < $< > $@
 
 EXTRA_DIST = \
-- 
2.30.0

